server/routes/utils/__tests__/savedSearchReportHelper.test.ts

```diff
@@ -330,15 +889,37 @@ describe('test create saved search report', () => {
 
   test('create report for data set with nested fields', async () => {
     const hits = [
-      hit({
-        'geoip.country_iso_code': 'GB',
-        'geoip.location': { lon: -0.1, lat: 51.5 },
-      }),
-      hit({
-        'geoip.country_iso_code': 'US',
-        'geoip.city_name': 'New York',
-        'geoip.location': { lon: -74, lat: 40.8 },
-      }),
+      hit(
+        {
+          'geoip.country_iso_code': 'GB',
+          'geoip.location': { lon: -0.1, lat: 51.5 },
+          customer_birth_date: '2023-04-26T04:34:32Z',
+          order_date: '2023-04-26T04:34:32Z',
+          products: { created_on: '2023-04-26T04:34:32Z' },
+        },
+        {
+          customer_birth_date: '2023-04-26T04:34:32Z',
+          order_date: '2023-04-26T04:34:32Z',
+          'products.created_on': '2023-04-26T04:34:32Z',
+        }
+      ),
+      // }),
```

Comment:
minor: can remove this comment and one line `922`

Reply:
done